index.html

```diff
@@ -0,0 +1,325 @@
+                    </div>
+                 
+                    <div class="main-expand-text">
+                        <div class="expand-text">
+                            <div class="inner-card">
+                                <img src="./images/js.png" alt="">
+                                <span>JavaScript</span>
+                            </div>  
+                        </div>
+                        <div class="expand-text">
+                            <div class="inner-card">
+                                <img src="./images/html.png" alt="">
+                                <span>HTML</span>
+                            </div>  
+                        </div>
+                        <div class="expand-text">
+                            <div class="inner-card">
+                                <img src="./images/css.png" alt="">
+                                <span>CSS</span>
+                            </div>  
+                        </div>
+                    </div>
+                   
+                   
+
+                </li>
+                <li>
+                    <div class="select-job">
+                        <a href="">
+                            Frameworks
+                       </a> 
+                       <i class="fa-solid fa-chevron-right"></i>
+                       <img src="images/right-arrow.png"/>
+                    </div>
+                
+                    
+                </li>
+                <li>
+                    <div class="select-job">
+                        <a href="">
+                            Skills
+                       </a> 
+                       <i class="fa-solid fa-chevron-right"></i>
+                       <img src="images/right-arrow.png"/>
+                    </div>
+                    
+                </li>
+            </ul>
+        </div>
+
+        
+    </section>
+    <footer>
+        <div class="form-container">
+            <div class="inner-form-container">
+                <div class="contact-container">
+                    <h1>Contact Me</h1>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                </div>
+                <div class="form-control">
+                    <form action="https://formspree.io/f/maykokbe" method="POST">
+                        <input type="text" name="fullname" maxlength="30" placeholder="Enter your fullname" required/>
+                        <input type="email" name="email" placeholder="Enter your email" required/>
+                        <textarea placeholder="Write your message here" name="Message" maxlength="500" required></textarea>
+                        <div class="btn-div">
+                            <button type="submit" class="submitBtn">Get in touch</button>
+
+                        </div>
+                    </form>
+                </div>
```

Comment:
Kindly fix the contact form to resemble the Figma design:

Your design:

![image](https://user-images.githubusercontent.com/73714615/192994223-ea7ad0bd-cbc2-4c11-b0e9-1644a3a34294.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192994264-9c840cbe-6c35-4bdb-a5f1-b9bea7665d0f.png)